solution/0000-0099/0078.Subsets/78. Subsets.js

```diff
@@ -0,0 +1,22 @@
+
+ //* @param {number[]} nums
+ //* @return {number[][]}
+ 
```

Comment:
Please keep the initial state.

```suggestion
/**
 * @param {number[]} nums
 * @return {number[][]}
 */
```